packages/cozy-harvest-lib/src/components/KonnectorConfiguration/ConfigurationTab/BiContractActivationWindow.jsx

```diff
@@ -49,7 +49,7 @@ const BIContractActivationWindow = ({
     if (konnectorPolicy.fetchExtraOAuthUrlParams) {
       handleLinkFetch()
     }
-  }, [konnector.slug, account, client, konnectorPolicy])
+  }, [konnector, account, client, konnectorPolicy])
```

Comment:
This konnector.slug was here on purpose. Or else you get multiple renders and multiple calls  to createTemporaryToken.

Reply:
Ah. The temporary token is stored in the konnector?